editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/adopters/configurators/MavenTemplateConfigurator.java

```diff
@@ -124,6 +124,10 @@ protected boolean configurePomCamelVersion(IProject project, NewProjectMetaData
 			
 			if(projectMetaData.getTargetRuntime() == null){
 				MavenUtils.alignFuseRuntimeVersion(m2m, camelVersion);
+			} else {
+				// we suppose that only one version of Fuse Runtime is possible for a Camel Version
+				//TODO: find a way to retrieve the Fuse Runtime BOM version from the Target Runtime
+				MavenUtils.alignFuseRuntimeVersion(m2m, camelVersion);
 			}
```

Comment:
I prefer to let it clear in the code that we rely on an assumption which is not perfect